tfhe/src/integer/server_key/radix_parallel/tests_signed/test_add.rs

```diff
@@ -556,7 +556,7 @@ where
     for num_blocks in 1..MAX_NB_CTXT {
         let modulus = (cks.parameters().message_modulus().0.pow(num_blocks as u32) / 2) as i64;
         if modulus == 1 {
-            // Basically have one bit the sign bit can't really test
+            // Basically have one bit, the sign bit can't really test
             continue;
         }
```

Comment:
Hum its not really clear when reading, bug, the `modulus` here is actually the `half_modulus` since its obtained by doing `modulus / 2`, so its the modulus ignoring the sign bit

So if `half_modulus == 1` then it means we have one bit which is the sign bit, as the 'value' bits only allow represent values with half_modulus == 1 so nothing

Reply:
are we able to test the minimum value then for a signed integer ?
i.e.

-2^b for some b ? as using the modulus on the abs value would limit us to -2^b + 1

Reply:
I will update the naming